show_menu.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'app'
+def show_menu_list
+  puts ''
+  puts 'Welcome to School Library App!'
+  puts ''
+  puts 'Choose an option by entering a number:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List rentals for a person'
+  puts '7. Quit'
+end
+
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] Kindly remove all `rubocop:disable` statements from this file. `Rubocop` helps us to write clean, readable and industry-standard codes. Using `rubocop:disable` statements defeats this purpose. Kindly remove it. 👍 💻